proto/evm/get_public_keys.proto

```diff
@@ -38,7 +38,6 @@ message GetPublicKeysIntiateRequest {
    */
   repeated GetPublicKeysDerivationPath derivation_paths = 2;
 
-  bool do_verify = 3;
```

Comment:
Revert this change for now, it's not related to solana. Should be taken up separately